compiler/rustc_passes/src/stability.rs

```diff
@@ -95,10 +96,12 @@ struct Annotator<'a, 'tcx> {
 impl<'a, 'tcx> Annotator<'a, 'tcx> {
     // Determine the stability for a node based on its attributes and inherited
     // stability. The stability is recorded in the index and used as the parent.
+    // fn_sig: Function signature linked with the current node
     fn annotate<F>(
         &mut self,
         hir_id: HirId,
         item_sp: Span,
+        fn_sig: Option<&hir::FnSig<'_>>,
```

Comment:
The pickiest of my nitpicks :innocent:: the HIR lives for `'tcx` which is the lifetime that's used everywhere in the compiler, can we make it explicit here?

```suggestion
        fn_sig: Option<&'tcx hir::FnSig<'tcx>>,
```

Reply:
Ahah, the compiler was smarter than me on this one 😄 Maybe the pickiest but I learned something through this